optapy-core/setup.py

```diff
@@ -52,8 +75,27 @@ def run(self):
         build_py.run(self)
 
 
+def find_stub_files(stub_root: str):
+    """
+    This function is taken from the awesome sqlalchey-stubs:
+    https://github.com/dropbox/sqlalchemy-stubs/blob/master/setup.py#L32
+    It's licensed under Apache 2.0:
+    https://github.com/dropbox/sqlalchemy-stubs/blob/master/LICENSE
+    """
```

Comment:
As much as I appreciate the spirit of fair play, this code is not rocket science. It is a fairly generic piece of code anyone could have written, and in fact probably has. I wouldn't bother with credit in this particular case.

Reply:
TBH, I copied it from another project, which copied it from that project (with the above comment). Referencing the source can help with context when re-reading the code in the future. Although the code is fairly simple.